mongodb/bucketarchives.go

```diff
@@ -36,7 +36,7 @@ type DealInfo struct {
 	Miner       string `bson:"miner"`
 
 	PieceCID string `bson:"piece_cid"`
-	Size     uint64
+	Size     uint64 `bson:"size"`
```

Comment:
I noticed this tag was missing. No problem because my understanding is the default bson behavior is to use the lowercase version of the property name, so nothing changes. Just did this for completeness.